### PR TITLE
Adding x509 attributes

### PR DESCRIPTION
**Pull request by ethanheppner**

### CODE REVIEW

**Comment by ethanheppner** [comment]
@achapm made the edits-- lmk if anything else is needed!

**Comment by ethanheppner** [comment]
@achapm are you good to approve and merge this?

File: _pages/attributes.md [hunk 1]
```diff
@@ -177,6 +177,64 @@ Requires the `profile` or `profile:verified_at` scope.
 </td>
 <td markdown="1">
 `verified_at` (string, ISO8601 format)
+</td>
+    </tr>
+    -----------------------------
```
Comment (zachmargolis) [inline _pages/attributes.md:182]: Should we remove this line? I'm not seeing it rendered

**Comment by zachmargolis** [comment]
On the OIDC page in the [User Info Response](https://developers.login.gov/oidc/#user-info-response) section we also document the attributes as they appear in OIDC, with some examples can we add these attributes there too?

File: _pages/attributes.md [hunk 2]
```diff
@@ -177,6 +177,64 @@ Requires the `profile` or `profile:verified_at` scope.
...
+**x509 Presented** <br />
+</td>
+<td markdown="1">
+![checkmark][checkmark]
+</td>
+<td markdown="1">
+![checkmark][checkmark]
+</td>
+<td markdown="1">
+`x509_presented` (string)
```
Comment (zachmargolis) [inline _pages/attributes.md:232]: I'm having trouble tracking down an example value for this -- we're sure it's a string right?

File: _pages/attributes.md (hunk 1 above)
Comment (ethanheppner) [reply _pages/attributes.md:182]: Yes, we can get rid of that-- I'd only put this in to keep my place and forgot to delete it afterwards.

File: _pages/attributes.md (hunk 2 above)
Comment (ethanheppner) [reply _pages/attributes.md:232]: I'm not sure-- I just added the content @achapm suggested, but if something else seems more correct to you, feel free to put that in.

**Pull request merged**